### ISSUE TITLE
IE9 fails to load minified shim

### ISSUE THREAD

**Issue by nmoinvaz** [paulmillr/es6-shim#449]
It appears that IE9 is having problems loading the minified shim. 
```
errorLine (11)
errorCharacter (5150)
errorCode (0)
errorMessage (Expected identifier, string or number)
errorUrl (es6-shim.min.js)
```
When I used the unminified version it works as expected. When I take the unminified version and minify it using Google's Closure Compiler, it works.

https://closure-compiler.appspot.com/

I believe the minfier that you are using is not turning the keyword `catch` for Promises into a string.

`...o.promise}});b(T,{catch:function(e){return this.then(nu...`

**Comment by ljharb** [comment]
Thanks, that sounds about right. I'll look into it.

**Comment by ljharb** [comment]
Fixed in 8583cbe2186ec12bfbf4218ccbce7fc73f96de87 (in the next release)
